apps/storefront/src/index.d.ts

```diff
@@ -40,6 +40,7 @@ declare interface Window {
     utils: {
       openPage: (page: import('./constants').HeadlessRoute) => void;
       getRoutes: () => import('@/shared/routeList').BuyerPortalRoute[];
+      setB3ConfigEventListeners: (key: string, value: string) => void;
```

Comment:
```suggestion
      setConfig: (key: string, value: string) => void;
```

we shouldn't expose `B3` as that is a reference to `bundleb2b` and we don't want that in our public api.

```javascript
b2b.utils.setConfig('dom.registerElement')
```